tests/test_ev_demand.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
```

Comment:
@alxsmith dumb question: where is the code the generates the accuracy metrics for the models? Maybe that should be a notebook or a script?

Reply:
a script in `utils/`